src/main/java/me/arynxd/button_utils/pagination/StandardButtonPaginator.java

```diff
@@ -41,7 +41,7 @@
     private final int maxPage;
     private final ActionRow actionRow;
     private final List<String> jwtTokens;
-    private long messageId = -1;
+    private long messageID = -1;
```

Comment:
This does not follow formatting guidelines or Java convention

```suggestion
    private long messageId = -1;
```